packages/core/admin/admin/src/pages/HomePage/ContentBlocks.js

```diff
@@ -19,6 +20,7 @@ const ContentBlocks = () => {
 
   return (
     <Stack spacing={5}>
+      <CloudBox />
```

Comment:
Does the `HomePage` component have tests? If so could you at least add one to confirm the component is rendered, please?